docs/sources/auth/overview.md

```diff
@@ -125,6 +125,10 @@ Defaults to `false`.
 oauth_auto_login = true
 ```
 
+If you want to skip automatic OAuth login and sign in with a username and password, then add the `disableAutoLogin` parameter to the login URL. This disables signing in with OAuth.
```

Comment:
Forget about version availability note. This feature will be available in 7.4.

Reply:
AH, yes. Definitely want to add that.